routes/things/delete.ts

```diff
@@ -0,0 +1,65 @@
+import { withRouteSpec } from "lib/middleware/with-winter-spec"
+import { z } from "zod"
+import type { DatabaseSchema } from "lib/db/schema"
+
+export default withRouteSpec({
+  methods: ["POST"],
+  formData: z.object({
+    thing_id: z.string(),
+  }),
+  jsonResponse: z.object({
+    ok: z.boolean(),
+  }),
+})(async (req, ctx) => {
+  let thing_id = ""
+
+  // Try to parse form data first
+  try {
+    const formData = await req.formData()
```

Comment:
use this

![image](https://github.com/user-attachments/assets/7a5596f5-2900-4690-956e-3dced88611c0)